docs/ocs-on-a-disconnected-environment.md

```diff
@@ -53,129 +53,157 @@ you should get something similar to this:
-registry.redhat.io/ocs4/mcg-core-rhel8@sha256:c717c19981446afc2c17460fa891c1a1fcb39be497b1bea3f86e87bb5e6c2305=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/ocs4/mcg-core-rhel8
-registry.redhat.io/ocs4/rhceph-rhel8@sha256:8f51e6b6ca3017eeebd57026fbf1f9f74831a18d13f605fac1b9694dd41e55cf=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/ocs4/rhceph-rhel8
-registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:797ff087ed64e38d0de12875f58ea395ec222bf67cad59a6abb6c131e12bfb97=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/openshift4/ose-csi-driver-registrar
-registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:72fcb2c114b7a4b0222802d4fe82ada7620162b1120462942c8a9d8e66230677=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/openshift4/ose-csi-external-attacher
-registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:203417dc512f0e3583be06db7f93a3729fef08d8e939ec1fc828ef914d1d8995=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/openshift4/ose-csi-external-provisioner-rhel7
-registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:ad5dc22e6115adc0d875f6d2eb44b2ba594d07330a600e67bf3de49e02cab5b0=yydev2.mirror-registry.qe.gcp.devcluster.openshift.com:5000/rhscl/mongodb-36-rhel7
-```
+below are examples of a mapping file for **OCS-4.2.0** and **OCS-4.3.0** (other versions will require different images. go over the CSV and make sure that all of the used images (can be grepped with `sha256`) are mapped in the file)  
+save the content below to `mapping.txt`  
+
+  for OCS-4.2.0:
+  ```
+  registry.redhat.io/ocs4/cephcsi-rhel8@sha256:280b39004dc8983eba810f6707b15da229efd62af512839a4f14efc983df2110=my.mirror-registry:5000/ocs4/cephcsi-rhel8
+  registry.redhat.io/ocs4/mcg-core-rhel8@sha256:c717c19981446afc2c17460fa891c1a1fcb39be497b1bea3f86e87bb5e6c2305=my.mirror-registry:5000/ocs4/mcg-core-rhel8
+  registry.redhat.io/ocs4/rhceph-rhel8@sha256:8f51e6b6ca3017eeebd57026fbf1f9f74831a18d13f605fac1b9694dd41e55cf=my.mirror-registry:5000/ocs4/rhceph-rhel8
+  registry.redhat.io/openshift4/ose-csi-driver-registrar@sha256:797ff087ed64e38d0de12875f58ea395ec222bf67cad59a6abb6c131e12bfb97=my.mirror-registry:5000/openshift4/ose-csi-driver-registrar
+  registry.redhat.io/openshift4/ose-csi-external-attacher@sha256:72fcb2c114b7a4b0222802d4fe82ada7620162b1120462942c8a9d8e66230677=my.mirror-registry:5000/openshift4/ose-csi-external-attacher
+  registry.redhat.io/openshift4/ose-csi-external-provisioner-rhel7@sha256:203417dc512f0e3583be06db7f93a3729fef08d8e939ec1fc828ef914d1d8995=my.mirror-registry:5000/openshift4/ose-csi-external-provisioner-rhel7
+  registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:ad5dc22e6115adc0d875f6d2eb44b2ba594d07330a600e67bf3de49e02cab5b0=my.mirror-registry:5000/rhscl/mongodb-36-rhel7
+  ```
+
+  for OCS-4.3.0:
```

Comment:
Need to have images at quay.io/noobaa added:
quay.io/noobaa/lib-bucket-catalog@sha256:b9c9431735cf34017b4ecb2b334c3956b2a2322ce31ac88b29b1e4faf6c7fe7d=<your_registry>/noobaa/lib-bucket-catalog
quay.io/noobaa/pause@sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105=<your_registry>/noobaa/pause

Reply:
Just note about the `quay.io/noobaa/pause` image - this image for the `lib-bucket-provisioner-...` pod is referenced without the sha256 digest (like `image: quay.io/noobaa/pause`), so the it will didn't try the mirror anyway and the pod will end up in `ImagePullBackOff` (or something like this) state.

Reply:
@netzzer why do we need the lib-bucket-catalog?